api/app/GraphQL/Mutations/TestPoolCandidateSnapshot.graphql

```diff
@@ -0,0 +1,180 @@
+                id
+                group
+                level
+                name {
+                  en
+                  fr
+                }
+              }
+              expectedGenericJobTitles {
+                id
+                key
+                name {
+                  en
+                  fr
+                }
+              }
+
+              isWoman
+              hasDisability
+              isIndigenous
+              isVisibleMinority
+              jobLookingStatus
+              hasDiploma
+              locationPreferences
+              locationExemptions
+              acceptedOperationalRequirements
+              expectedSalary
+              expectedClassifications {
+                id
+                name {
+                  en
+                  fr
+                }
+                group
+                level
+              }
+              wouldAcceptTemporary
+              cmoAssets {
+                id
+                name {
+                  en
+                  fr
+                }
+                description {
+                  en
+                  fr
+                }
+                key
+              }
+              experiences {
+                id
+                __typename
+                applicant {
+                  id
+                  email
+                }
+                details
+                skills {
+                  id
+                  key
+                  name {
+                    en
+                    fr
+                  }
+                  description {
+                    en
+                    fr
+                  }
+                  keywords {
+                    en
+                    fr
+                  }
+                  experienceSkillRecord {
+                    details
+                  }
+                }
+                ... on AwardExperience {
+                  title
+                  issuedBy
+                  awardedDate
+                  awardedTo
+                  awardedScope
+                }
+                ... on CommunityExperience {
+                  title
+                  organization
+                  project
+                  startDate
+                  endDate
+                }
+                ... on EducationExperience {
+                  institution
+                  areaOfStudy
+                  thesisTitle
+                  startDate
+                  endDate
+                  type
+                  status
+                }
+                ... on PersonalExperience {
+                  title
+                  description
+                  startDate
+                  endDate
+                }
+                ... on WorkExperience {
+                  role
+                  organization
+                  division
+                  startDate
+                  endDate
+                }
+              }
+              # awardExperiences {
```

Comment:
Any reason why these are commented out or can they be removed?

Reply:
I literally copied this from frontend/talentsearch/src/js/api/userOperations.graphql and I'd rather not make any cosmetic changes to the query that create noise when comparing the two files since it's super important that they stay in sync.  The only change I made was to change it from "me" to a specific user.

Reply:
I wonder if there's a way to make sure the same query is used between the frontend and api without introducing a dependency.  :thinking:

Reply:
Ok, I see. Not sure if it's possible, but what about creating a fragment and importing it? Not sure if urql can do it but Apollo certainly [does](https://www.apollographql.com/docs/react/data/fragments/).

I found the [origin](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2369/files#diff-200ddc7e7d0a5cc02eab37e0bf1f102c8757983d7c7036f576b6204d636db9ff) of the commented out code. I know this was ages ago, but any idea why the commented code was added @vd1992?

Reply:
I think that was back when experiences were not quite ready to be loaded, so it was commented out for later
instead a separate block was added in for experiences and the commented out stuff left

Reply:
Thanks! @vd1992 can the commented stuff be removed then? If not, ignore my comments please @petertgiles. If so, can you remove from both places to cut our debt and future questions regarding the unused code?

Reply:
don't see why not

Reply:
Queries tidied in [1493a06](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3707/commits/1493a062107331594d6283d4a016c878402c7ecf)

Reply:
Spun out query consolidation to #3713